spec/.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+inherit_from: '../.rubocop.yml'
```

Comment:
Interesting. What do you think are the pros/cons in using multiple `rubocop` config files over a single one?

Reply:
**Actual answer to your question.**

In my experience you need at least two rubocop.yml files.
One for your `app` and `lib` folder - to determine the style for your code.
Another one for your `spec` folder - to detirminet the style for your tests.
Tests usually have different style requirements than regular code, especially 
when using  RSpec's DSL.

Having more config files than this leads to inconsistencies, and is a bad practice in my opinion.

**Ramblings about `rubocop-rspec`**

I'm pretty sure `rubocop-rspec` automatically does this - it loads it's own rubocop config for the specs.
I didn't use `rubocop-rspec` because it is too strict for my taste.
I've used it on previous professional projects, but spent a lot of time fighting it.

I'm sure you can learn to benefit from it's warnings, but it distracts me from what I am trying to
test, and how I want to logically structure my tests. And the fact that I have to worry about my setup
functions beeing under a specific line length is mostly a hindrance, and not a help to improve my specs.

**Legacy code woes**

One reason to have extra rubocop config files is when working with legacy code.
Basically you could namespace your legacy code into a `Legacy` namespace, and apply
a different, far more lenient rubocop config, while your new code uses a better rubocop config.

Trying to refactor legacy code, because Rubocop complains is a terrible idea in my opinion, because
you loose valuable git history, as you overwrite the lines from the previous commits.
You can still dig through your git history, to see what is the context of the commit previous to the one
that refactored for Rubocop's sake, but that is alot of extra hassle.